style.css

```diff
@@ -0,0 +1,87 @@
+.burger {
+	background-color: salmon;
+	display: none;
+	height: 3px;
+	position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/CSS17K/pullRequest?prid=5818552)